portworx-manual/Portworx.md

```diff
@@ -110,10 +127,65 @@ Volume  :  924341013124639820
                   Running on     : aks-nodepool1-13284751-1
                   Controlled by  : kafka (StatefulSet)
 ```
-For details on how to snapshot volumes using stork check out - https://portworx.com/run-ha-kafka-azure-kubernetes-service/.
 
 ## Monitoring Portworx
 
-In the provided Portworx spec file, `px-gen-spec.yaml`, there is an added port opening configuration under the Daemonset section allowing Prometheus to scrape Portworx pods for metrics. In addition, added annotations for promethes to access the metric probes have been embedded for Prometheus integration. Navigate to prometheus and view the `px` metrics.
+In the provided Portworx spec file, `px-gen-spec.yaml`, there is an added port opening configuration under the daemonset section allowing Prometheus to scrape Portworx pods for metrics. In addition, added annotations for Prometheus to access the metric probes have been embedded for Prometheus integration. Navigate to prometheus and view the `px` metrics.
 
-> Run `kubectl port-forward -n prometheus <prometheus_server_pod> 9090`
\ No newline at end of file
+> Run `kubectl port-forward -n prometheus <prometheus_server_pod> 9090`
+
+## High Availability & Snapshots with Portworx
+
+High Availability and Disaster Recovery are built into Portworx out of the box. Here are some tips and 
+
+### Portworx Failovers
+
+You can simulate the failover for Portworx by cordoning off one of the nodes and deleting the Kafka Pod deployed on it. To start, follow the steps belor or run the [`px-failovertest.sh`](temp-px-perf/px-failovertest.sh). When the new pod is created it has the same data as the original Pod.
```

Comment:
Perhaps [explain](https://kubernetes.io/docs/concepts/architecture/nodes/#manual-node-administration) that `kubectl cordon` will prevent new pods from being scheduled on the node